pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -770,7 +770,7 @@ func (m *ManagerImpl) callPreStartContainerIfNeeded(podUID, contName, resource s
 // the node to pass GeneralPredicates admission checking even upon device plugin failure.
 func (m *ManagerImpl) sanitizeNodeAllocatable(node *schedulercache.NodeInfo) {
 	var newAllocatableResource *schedulercache.Resource
-	allocatableResource := node.AllocatableResource()
+	allocatableResource := node.AllocatableResource
```

Comment:
I think there's no need to change how other programs are using our `NodeInfo` type in this PR. It will be significantly easier to merge this if we do not make changes outside of scheduling code.

Reply:
If don't change NodeInfo, all the fields in the structure are private, and it is very cumbersome and complicated to modify and use externally.

Reply:
For example, we need to implement a lot of Set*() functions.